paper.tex

```diff
@@ -151,36 +146,38 @@ \section*{Introduction}
 principles, but share some common properties. 
 Firstly, they are all 
 hueristic and approximation driven, favouring computational
-scalability over statistical rigor.
+scalability over a basis in a statistical model.
 Secondly, they all produce a single
 ARG as output, inferring a deterministic point estimate with no means 
 of quantifying uncertainty. Thirdly (and most importantly for the 
-purposes of this paper) the ARGs they produce lack substantial
-detail about ancestral recombination events: the estimates are
-rough sketches, hopefully capturing large-scale properties, but lacking
-in the finer detail of outputs of ARGweaver or SINGER. 
+purposes of this paper) the ARGs they produce 
+lack some of the information present in ARGweaver or SINGER. 
+This occurs, roughly speaking, because of a fundamental difference in approach:
+not all details of the ARG can be reliably inferred
+(e.g., the timing of recombination events, as discussed below);
+and the heuristic methods deal with this uncertainty by omitting these events,
+while ARGweaver and SINGER provide a sample from the posterior.
+(So, some aspects of the resulting ARGs should be interpreted slightly differently;
+see \citet{wong_general_2023} for more discussion.)
 
 There are substantial advantages to the statistically rigorous sampling 
-approaches of ARGweaver and SINGER. Firstly, they tend to be more 
-accurate; ARGweaver has consistently out-performed other methods in terms 
-of accuracy in a variety of benchmarks~\citep{brandt2022evaluation}[MORE
-citations,anywhere ARGweaver has been tested for accuracy]
-and, while SINGER has not been evaluated as thoroughly yet, it promises
-to be even more accurate~\citep{deng_robust_2024}. 
-Secondly, the direct connection with
-population genetic models greatly aids interpretability, and allows for
+approaches of ARGweaver and SINGER.
+Firstly, they tend to be more accurate when compared on the same data set.
+Secondly, the direct connection with population genetic models allows for
 natural extensions like the joint estimation of ARGs and
-demography~\citep{hubisz_mapping_2020}. Thirdly, sampling ARGs from a
+demography~\citep{hubisz_mapping_2020}.
+Thirdly, sampling ARGs from a
 posterior distribution provides a means of quantifying the uncertainty 
-around estimates, which can be used in downstream applications [citation
-of this?]. 
+around estimates, which can be used in downstream applications.
 
-Despite these significant advantages of rigorous inference, however,
-hueristic methods remain of interest because of the profound difference
-in computational scalability. Although SINGER is a major step forward
+On the other hand, heuristic methods have the potential
+to utilize the greater amount of information in large datasets,
+thus possibly producing even more accurate estimates
+(although this has not, to our knowledge, been tested).
```

Comment:
we may want to cut this note, but I think it's a good point

Reply:
I agree, but will probably cut it as it may invite some unwanted back-and-forth in review